README.md

```diff
@@ -199,6 +199,7 @@ search.addWidget(
     container: '#search-box',
     placeholder: 'Search for products',
     // cssClass
+    // poweredBy: boolean
```

Comment:
@pixelastic 
In the end, we didn't agree on this one in the issue. In the end, the default value should probably make more sense in integrations, and we can put it ourselves. Open to discussion on this one.

@redox @vvo @bobylito ?